Group 5 of Group 2 Review on 20200325

skjohansen opened the pull request.

murkug committed:
initial commit

@erifredrika @adgnascor @irvper77 @muratkugic: initial confused commit

murkug committed:
Added Database

@murakugic @erifredrika @adgnascor @irvper77 added database and relationships code first

Commit:
@adgnascor

Added my own environment and connectionstring

Commit:
@adgnascor

Added StarwarsShip request to API

First try, to figure out how it works

erifredrika committed:
added people api

Added classes and stuff so we can get information out of the api

erifredrika committed:
Merge branch 'Dev' of https://github.com/PGBSNH19/the-spaceport-group-2 into Dev

Adgnascor committed:
Delete NewFile2.txt

erifredrika committed:
added name

Name

erifredrika committed:
Added if/else and cool header

Added some logic, perhaps temporary. We could of course do it more neatly I guess.

Also added a super cool header yo

erifredrika committed:
Minor changes

Added toLower

erifredrika committed:
Updated database stuff

Updated database tables

erifredrika committed:
added a table

added table (everyone)

Co-Authored-By: William Lindahl <adgnascor@users.noreply.github.com>
Co-Authored-By: irvper77 <irvper77@users.noreply.github.com>
Co-Authored-By: Murat <muratkugic@users.noreply.github.com>

erifredrika committed:
hej

test merge

erifredrika committed:
Merge branch 'Fredrika' into Dev

murkug committed:
changed connectionstring

murkug committed:
spring cleaning

murkug committed:
trying to get figure out who parked where

murkug committed:
testing to see if my theory holds

murkug committed:
This seems to work satifactory..

I can now stear how many available spaces I have in the garage and it adjusts accordingly when it is full -> program exits.

murkug committed:
.

murkug committed:
Menu working

murkug committed:
improved menu

murkug committed:
cleaned up code

murkug committed:
I can now retrieve a name from the database who is currently occupying a parkinglot

murkug committed:
.

murkug committed:
spring cleaning before code review

@erifredrika @adgnascor @irvper77 @muratkugic

Bluemace commented on Source/SpacePark/Program.cs:
Kan använda en exception för not yet implemented här istället för att ha "placeholder" kod.

Bluemace commented on Source/SpacePark.Library/Models/StarwarsAPI.cs:
Vi är osäkra på vad detta uppfyller för funktion, vi säger inte att det är rätt eller fel, men vi förstår inte vad detta är till för.

Bluemace reviewed (commented):
Överlag så tyckte vi att koden var lite svår att hänga med i och att det var lite rörigt, vi förstår att koden inte är färdig och att det kan vara därför som det är rörigt. Programmet saknar även unittester

erifredrika committed:
delete update

delete update

erifredrika committed:
update

update

murkug committed:
.

murkug committed:
.

murkug committed:
.

erifredrika committed:
baseapp working

works, kind of

Co-Authored-By: William Lindahl <adgnascor@users.noreply.github.com>
Co-Authored-By: Murat <muratkugic@users.noreply.github.com>
Co-Authored-By: irvper77 <irvper77@users.noreply.github.com>

murkug committed:
Merge branch 'newnewDev' into Murat

Commit:
@adgnascor

Adgnascor committed:
Deleted appsettings

Commit:
@adgnascor

Test if appsetting.json still exist in repo

Commit:
@adgnascor  Added to GitIgnore

Adgnascor committed:
Delete appsettings.json

Adgnascor committed:
Update .gitignore

Commit:
@adgnascor  Fighting With GitIgnore

Commit:
Merge branch 'William_Dev' of https://github.com/PGBSNH19/the-spaceport-group-2 into William_Dev

Commit:
@adgnascor Removed comment

Commit:
@adgnascor  Fixed Payment problem

Fixed problem when trying to pay after relaunching the application. Before it crashed, now it work when entering correct name

murkug committed:
delete file since it only need to be on local

murkug committed:
deleted empty file

murkug committed:
.

Commit:
@adgnascor

Full Working (But simple) application

murkug committed:
.

murkug committed:
Merge branch 'master' into Murat

Commit:
@adgnascor

Trying to fix problem when entering wrong name or empty name!

murkug committed:
Merge branch 'master' into Murat

murkug committed:
error checking when entering name and ship

Co-Authored-By: Fredrika Eriksson <erifredrika@users.noreply.github.com>
Co-Authored-By: William Lindahl <adgnascor@users.noreply.github.com>
Co-Authored-By: irvper77 <irvper77@users.noreply.github.com>

murkug committed:
Added tests

Co-Authored-By: Fredrika Eriksson <erifredrika@users.noreply.github.com>
Co-Authored-By: William Lindahl <adgnascor@users.noreply.github.com>
Co-Authored-By: irvper77 <irvper77@users.noreply.github.com>
